### PR TITLE
Added Online Banking PL (P24) component

### PR DESCRIPTION
**Pull request by jobsturm**
## Summary
This adds support for Online Banking PL (P24)

### CODE REVIEW

File: packages/playground/src/pages/IssuerLists/IssuerLists.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ import '../../style.scss';
     const session = await createSession({
         amount: {
             value: 123,
-            currency: 'EUR'
```
Comment (ribeiroguilherme) [inline packages/playground/src/pages/IssuerLists/IssuerLists.js:12]: We can leave 'EUR' as default - there is no point to change it

**Review by ribeiroguilherme — APPROVED** [review]

File: packages/lib/src/components/P24/index.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import IssuerListContainer from '../helpers/IssuerListContainer';
+
+class P24Element extends IssuerListContainer {
```
Comment (ribeiroguilherme) [inline packages/lib/src/components/P24/index.ts:3]: Better to name the payment method to `OnlineBankingPL`, same as we do with the other online banking payment methods (`OnlineBankingCZ`, `OnlineBankingSK`..).  Also rename the folder that keeps it.

File: packages/lib/src/components/P24/index.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+    public static type = 'dotpay';
```
Comment (ribeiroguilherme) [inline packages/lib/src/components/P24/index.ts:4]: Are we sure that tx variant is `dotpay` ? It seems that the ticket description mentions `onlineBanking_PL`

File: packages/playground/src/pages/IssuerLists/IssuerLists.js [hunk 4]
```diff
@@ -48,6 +48,9 @@ import '../../style.scss';
     // Dotpay
     window.dotpay = checkout.create('dotpay').mount('.dotpay-field');
 
+    // P24
+    window.dotpay = checkout.create('p24').mount('.p24-field');
```
Comment (ribeiroguilherme) [inline packages/playground/src/pages/IssuerLists/IssuerLists.js:52]: This variable name conflicts with the one on line 49.

**Review by ribeiroguilherme — CHANGES_REQUESTED** [review]

File: packages/lib/src/components/index.ts [hunk 5]
```diff
@@ -131,6 +132,7 @@ const componentsMap = {
     molpay_ebanking_VN: MolPayEBankingVN,
     onlinebanking_IN: OnlineBankingINElement,
     openbanking_UK: OpenBankingUK,
+    p24: P24,
```
Comment (ribeiroguilherme) [inline packages/lib/src/components/index.ts:135]: This mapping should be updated as well to use `onlineBanking_PL` key

**Review by ribeiroguilherme — DISMISSED** [review]

File: packages/lib/src/components/UIElement.tsx [hunk 6]
```diff
@@ -228,6 +228,7 @@ export class UIElement<P extends UIElementProps = any> extends BaseElement<P> im
      * Return the type of an element
      */
     get type(): string {
+        console.log(this.props.type || this.constructor['type']);
```
Comment (sponglord) [inline packages/lib/src/components/UIElement.tsx:231]: Can we get rid of this ;-)

**Review by sponglord — APPROVED** [review]

**Pull request merged**